package.json

```diff
@@ -8,19 +8,19 @@
     "client",
     "javascript"
   ],
-  "version": "0.2.0",
+  "version": "0.2.1",
   "homepage": "https://github.com/feedhenry/openstack-storage",
   "repository": {
     "type": "git",
     "url": "git://github.com/feedhenry/openstack-storage.git"
   },
   "main": "lib/storage.js",
   "dependencies": {
-    "optimist": "*",
-    "winston": "*",
-    "dateformat": "*",
     "async": "0.1.18",
-    "request": "2.9.202"
+    "dateformat": "*",
```

Comment:
@camilamacedo86 should we tie down this & the `optimist` and `winston` dependencies below to a specific major version version rather than any available version (`"*"`)

Reply:
Done 👍